### PR TITLE
TGUI input box conversions 1

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
Converts a batch of input boxes into TGUI
Cleans up some of the code surrounding

tgui_input_text changes:
- stripped_input is now the default behavior of tgui_input_text, thus html_encoding (and sanitizing) input
- small changes to sizing in tgui_input_text windows, they now fit more accurately

tgui_input_number changes:
- sizing changes same as above
- bug fixes with min/max values

## Why It's Good For The Game
- tgui input boxes are already there
- new flags for input sanitization
- smaller batch conversions for easier troubleshooting

## Changelog

:cl:
refactor: Several input prompts have been converted to TGUI. Nukie prompts, newscasters, stacks of materials, hand labelers, station charters. Check them out!
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/vote.dm [hunk 1]
```diff
@@ -188,14 +188,14 @@ SUBSYSTEM_DEF(vote)
 				for(var/valid_map in maps)
 					choices.Add(valid_map)
 			if("custom")
-				question = stripped_input(usr,"What is the vote for?")
+				question = tgui_input_text(usr, "What is the vote for?", "Custom Vote", max_length = MAX_MESSAGE_LEN)
```
Comment (optimumtact) [inline code/controllers/subsystem/vote.dm:191]: does this have the exact same stripping semantics as stripped_input?

Comment (jlsnow301) [reply code/controllers/subsystem/vote.dm]: going to html_encode all tgui text input by default, but you can simply ```encode = FALSE``` this behavior. so to answer your question, yes, it does now

**Comment by jlsnow301** [comment]
Adding no_trim but I can't find any actual uses of it

File: code/modules/tgui/tgui_input_text.dm [hunk 2]
```diff
@@ -50,10 +52,12 @@
  * * default - The default (or current) value, shown as a placeholder.
  * * max_length - Specifies a max length for input.
  * * multiline -  Bool that determines if the input box is much larger. Good for large messages, laws, etc.
+ * * encode - If toggled, input is filtered via html_encode. Setting this to FALSE gives raw input.
+ * * no_trim - Like stripped_input's no_trim. Boolean determines whether to use copytext or trim
```
Comment (Mothblocks) [inline code/modules/tgui/tgui_input_text.dm:56]: https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#avoid-negative-variable-names

Change to something that reads positively, like `use_copytext`, or better yet, change it to something like `bikeshed = BIKESHED_COPYTEXT` or `bikeshed = BIKESHED_TRIM`.

Also explain why you might want this.

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Pull request merged**